.travis.yml

```diff
@@ -18,7 +18,7 @@ install:
   - opam pin add --no-action ocamlformat .
   - opam install --deps-only ocamlformat
 script:
-  - ! git diff --exit-code "$TRAVIS_MERGE_BASE..$TRAVIS_PR_HEAD" -- CHANGES.md >/dev/null
+  - if [ $TRAVIS_BRANCH = master ]; then ! git diff --exit-code $TRAVIS_BRANCH...$TRAVIS_COMMIT -- CHANGES.md >/dev/null; fi
```

Comment:
The build running after a PR is merged is failing, maybe we should also test `TRAVIS_PULL_REQUEST != false` to only check the changelog for PRs ?

Reply:
Yes we should do that. I'll make a PR